chainstate/test-framework/src/random_tx_maker.rs

```diff
@@ -83,21 +134,28 @@ impl<'a> RandomTxMaker<'a> {
     pub fn make(
         mut self,
         rng: &mut (impl Rng + CryptoRng),
-    ) -> (Transaction, TokensAccountingDeltaData) {
+    ) -> (
+        Transaction,
+        TokensAccountingDeltaData,
+        PoSAccountingDeltaData,
+    ) {
         // TODO: ideally all inputs/outputs should be shuffled but it would mess up with token issuance
```

Comment:
Even though this looks like a small issue, it could have a large effect on randomization. Perhaps you should come back to that when you're done with reorg tests.

Reply:
ok I'll think what I can do there later